gluten-ut/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenLiteralExpressionSuite.scala

```diff
@@ -20,8 +20,10 @@ package org.apache.spark.sql.catalyst.expressions
 import org.apache.spark.sql.{GlutenTestConstants, GlutenTestsTrait}
 
 class GlutenLiteralExpressionSuite extends LiteralExpressionSuite with GlutenTestsTrait {
-
-  override def blackTestNameList: Seq[String] = Seq(
-    GlutenTestConstants.IGNORE_ALL
-  )
+  override def blackTestNameList: Seq[String] = {
+    val backendType = System.getProperty("backend_type", "")
```

Comment:
https://github.com/oap-project/gluten/pull/415 you can use not support suite interface